src/EditorFeatures/Core/EditorFeaturesResources.resx

```diff
@@ -730,4 +730,10 @@ Do you want to proceed?</value>
   <data name="TheRenameTrackingSessionWasCancelledAndIsNoLongerAvailable" xml:space="preserve">
     <value>The rename tracking session was cancelled and is no longer available.</value>
   </data>
+  <data name="Rename2" xml:space="preserve">
+    <value>Rename '{0}'</value>
+  </data>
+  <data name="SingleQuote" xml:space="preserve">
```

Comment:
Consider making two resources, ``` RenameDashboardTitleLeftOfIdentifier ``` and ``` RenameDashboardTitleRightOfIdentifier ``` with notes that explain to the localizers how they are used together.

Alternatively, we could consider going to a number-of-characters limitation rather than width limitation. I actually prefer the width limitation, it just introduces this extra localization challenge.